src/BridgeMaker.js

```diff
@@ -7,7 +9,17 @@ const BridgeMaker = {
    * @param {function(): number} generateRandomNumber 무작위 값을 생성해주는 함수
    * @return {string[]} 입력받은 길이에 해당하는 다리 모양. 위 칸이면 U, 아래 칸이면 D로 표현해야 한다.
    */
-  makeBridge(size, generateRandomNumber) {},
+  makeBridge(size, generateRandomNumber) {
+    const canWalkBridge = [];
+
+    while (canWalkBridge.length < size) {
+      const number = generateRandomNumber();
+      number === GAME.LOWER_BRIDGE_NUMBER
+        ? canWalkBridge.push(GAME.LOWER_BRIDGE_STRING)
+        : canWalkBridge.push(GAME.UPPER_BRIDGE_STRING);
+    }
```

Comment:
코드상에서 1은 true, 0은 false로 인식하는걸로 알고 있어서 
`MAKER.LOWER_BRIDGE_NUMBER`를 따로 생성해서 사용하지 않고 
`generateRandomNumber()`객체에서 생성된 0과 1 값으로 true/false를 판단해서 삼항연산자를 사용해보시는건 어떠실까요?😊

이렇게 작성하면 중복 코드를 조금 더 줄여볼 수 있을 것으로 기대된다고 생각합니다 :)

```suggestion
		while (canWalkBridge.length < size) {
			const number = generateRandomNumber();
			canWalkBridge.push(number ? COMMAND.UPPER_BRIDGE_STRING : COMMAND.LOWER_BRIDGE_STRING);
		}
```